tests/ui/single_component_use_path.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0254]: the name `semver` is defined multiple times
+  --> $DIR/single_component_use_path.rs:5:5
+   |
+LL | extern crate semver;
+   | -------------------- previous import of the extern crate `semver` here
+LL | use semver;
+   |     ^^^^^^ `semver` reimported here
+   |
+   = note: `semver` must be defined only once in the type namespace of this module
```

Comment:
A lint is provided by rust? Do we need to proceed?

Reply:
`extern crate ident` is conflict with `use ident`, is there a way to use crate without writing `extern crate ident`, or should I fall back to use std?

Reply:
Huh, you can use `semver::..` directly in edition 2015, but when you want to use it in a `use` statement you have to use `extern crate` first. 

Yeah, so I think gating it on edition 2018 would be better. Sorry for the back and forth.

Reply:
I'll try that out then.